calibration_rate:           1.0
ground_frame:               base_footprint

input_max_noise:              0.04
input_threshold_from_ground:  0.15
input_max_nan_ratio:          0.5
input_max_zero_ratio:         0.5
input_min_data_ratio:         0.7

max_deviation_degrees:  6.0
iterations:             4

precompute_planes:              true
precomputed_plane_pairs_count:  40

plane_max_too_low_ratio:  0.02
plane_max_mean:           0.03

debug:          false
always_update:  false

maximum_range_of_depth_camera: 3.5
threshold_normalized_z_by_xy: 0.16
threshold_normalized_invalid_z_by_xy: 0.32
max_angle_change: 0.8
  
